TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/ChangeLog.cshtml

```diff
@@ -11,4 +11,40 @@
     ViewBag.Title = Model.Name;
 }
 
-<p class="govuk-body">Change log goes here</p>
+@if (Model.TimelineItems is null || Model.TimelineItems.Length == 0)
+{
+    <p class="govuk-body" data-testid="no-changes">No changes</p>
+}
+else
+{
+    var itemNumber = 0;
+    <div class="moj-timeline">
+        @foreach (var timelineItem in Model.TimelineItems)
+        {
+            itemNumber++;
+            <div class="moj-timeline__item govuk-!-padding-bottom-2" data-testid="timeline-item">
+                <div class="moj-timeline__header">
+                    <h2 class="moj-timeline__title" data-testid="timeline-item-title">@timelineItem.Title</h2>
+                    <p class="moj-timeline__byline" data-testid="timeline-item-user">by @timelineItem.User</p>
+                    @if (timelineItem.Status is not null)
+                    {
+                        var tagClass = timelineItem.Status switch
+                        {
+                            ChangeLogModel.TimelineItemStatus.Overdue => "govuk-tag--red",
+                            ChangeLogModel.TimelineItemStatus.Closed => "govuk-tag--grey",
+                            _ => "govuk-tag"
```

Comment:
I think the tag helper will add `govuk-tag` already
```suggestion
                            _ => ""
```

Reply:
Done